feature/platform/transceiver/zr_tunable_parameters_test/README.md

```diff
@@ -0,0 +1,137 @@
+# TRANSCEIVER-5: Configuration: 400ZR channel frequency and output TX launch power setting. 
+
+## Summary
+
+Validate setting 400ZR tunable parameters channel frequency and output TX
+launch power and verify corresponding telemetry values.
+
+### Goals
+* Verify full C band frequency tunability for 100GHz line system grid.
+* Verify full C band frequency tunability for 75GHz line system grid.
+* Verify adjustable range of transmit output power across -13 to -9 dBm
+  in steps of 1 dB.  
+
+
+## TRANSCEIVER-5.1
+
+*   Connect two ZR interfaces using a duplex LC fiber jumper such that TX
+    output power of one is the RX input power of the other module. Connection
+    between the modules should pass through an optical switch that can be
+    controlled through automation to simulate a fiber cut.  
+*   To establish a point to point ZR link ensure the following:
+      * Both transceivers states are enabled.
+      * Validate setting 400ZR optics module tunable laser center frequency
+        across frequency range 196.100 - 191.400 THz for 100GHz grid.
+      * Validate setting 400ZR optics module tunable laser center frequency
+        across frequency range 196.100 - 191.375 THz for 75GHz grid.  
+      * Specific frequency details can be found in 400ZR implementation
+        agreement under sections 15.1 ad 15.2 Operating frequency channel
+        definitions.
+      * Validate adjustable range of transmit output power across -13 to -9 dBm
+        range in steps of 1dB. So the module’s output power will be set to -13,
+        -12, -11, -10, -9 dBm in each step.
```

Comment:
Should this be done just for one frequency?  Does it matter which frequency this is done at?

Reply:
Added a note this can be validated for modules default frequency of 193.1 Thz